skiko/src/awtTest/kotlin/org/jetbrains/skiko/SkiaSwingLayerTest.kt

```diff
@@ -0,0 +1,87 @@
+package org.jetbrains.skiko
+
+import kotlinx.coroutines.delay
+import org.jetbrains.skia.Canvas
+import org.jetbrains.skia.Paint
+import org.jetbrains.skia.Rect
+import org.jetbrains.skiko.swing.SkiaSwingLayer
+import org.jetbrains.skiko.util.ScreenshotTestRule
+import org.jetbrains.skiko.util.uiTest
+import org.junit.Rule
+import java.awt.Color
+import java.awt.Dimension
+import java.awt.Graphics
+import java.awt.Graphics2D
+import java.awt.RenderingHints
+import javax.swing.JComponent
+import javax.swing.JFrame
+import javax.swing.JLayeredPane
+import javax.swing.WindowConstants
+import kotlin.test.Test
+
+@OptIn(ExperimentalSkikoApi::class)
+class SkiaSwingLayerTest {
+
+    @get:Rule
+    val screenshots = ScreenshotTestRule()
+
+    @Test
+    fun `overlapped popup`() = uiTest {
```

Comment:
Worth to check also the case `overlapped skia layer`, when we draw skia content over the swing content.

Reply:
Added!